applications/zpc/components/zwave_command_handler/scripts/extract_get.py

```diff
@@ -3,7 +3,7 @@
 # Copyright (c) 2014 Silicon Laboratories Inc.
 
 from __future__ import print_function
-import xml.dom.minidom
+from defusedxml import minidom
 import sys
 import os.path
 
```

Comment:
nipticking(non-blocking): organize the import (https://peps.python.org/pep-0008/#imports).

```suggestion

from defusedxml import minidom
```